### PR TITLE
Feature/make coc static page

### LINKED ISSUE #39: Make Code of Conduct a static page
Use kramdown.
https://github.com/slim-template/slim#embedded-engines-markdown-

### PR DESCRIPTION
**Pull request by azi**
close #39

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -9,9 +9,15 @@ gem "sass-rails", "~> 5.0"
 gem "jquery-rails"
 gem "turbolinks"
 gem "jbuilder", "~> 2.0"
-gem "slim-rails"
 gem "rails_utils"
 
+# slim, markdown
+gem "slim-rails"
+gem "redcarpet"
```
Comment (AnNOtis) [inline Gemfile]: What do you think about including only one of these markdown parsers?

Comment (JuanitoFatas) [inline Gemfile]: kramdown 👍🏻

Comment (azi) [inline Gemfile]: just follow by issue requirement  :see_no_evil: :camel: :sheep: :sheep:

File: Gemfile [hunk 2]
```diff
@@ -9,9 +9,12 @@ gem "sass-rails", "~> 5.0"
 gem "jquery-rails"
 gem "turbolinks"
 gem "jbuilder", "~> 2.0"
-gem "slim-rails"
 gem "rails_utils"
 
+# slim, markdown
+gem "slim-rails"
+gem "kramdown"
```
Comment (JuanitoFatas) [inline Gemfile:16]: @azi only use kramdown is enough :grinning:

File: app/views/home/code_of_conduct.html.slim [hunk 3]
```diff
@@ -0,0 +1,24 @@
+.col-md-8.col-md-offset-2
```
Comment (JuanitoFatas) [inline app/views/home/code_of_conduct.html.slim:1]: @azi :point_right: I centralize it.

**Comment by JuanitoFatas** [comment]
@azi Thank you so much for your work!!! :+1:

File: Gemfile (hunk 1 above)
Comment (azi) [inline Gemfile]: sorry @AnNOtis   , I got wrong meaning about your question  :sweat: :pray: :stuck_out_tongue_closed_eyes:

before I used markdown in slim file that show me error message said that "not such file 'redcarpet'"
so I thought I need redcarpet....but actually don't need  ha
thanks your question  :+1:

**Comment by azi** [comment]
@JuanitoFatas   Thanks your review :bowtie:  :smiley_cat::smiley_cat::smiley_cat:

File: Gemfile (hunk 1 above)
Comment (AnNOtis) [inline Gemfile]: It's okay. You did a great job. :smiley:

**Pull request closed without merging**